lib/manager/npm/post-update/lerna.ts

```diff
@@ -82,18 +93,7 @@ export async function generateLockFiles(
       execOptions.docker.volumes = [[homeNpmrc, '/home/ubuntu/.npmrc']];
     }
     cmd.push(`${lernaClient} install ${cmdOptions}`);
-    let lernaVersion: string;
-    try {
-      const pJson = JSON.parse(await getFile('package.json'));
```

Comment:
I see one new issue now. What if lerna dep is only added to root `package.json` as dependency? Will that be a supported scenario? If yes we won't find lerna version any more.

Reply:
That's how it should be setup, right? Only the root package should depend on lerna? And that root package _is_ the `lernaPackageFile` we pass above, or am I missing sthg?

Reply:
![image](https://user-images.githubusercontent.com/6311784/86776868-d87dce80-c058-11ea-8269-fe7cf39a4c0d.png)
^^ this is the root `package.json`, right?

Reply:
root as far as Lerna is concerned yes, but may not be in the root directory of the git repo